src/HEREMap.tsx

```diff
@@ -18,6 +20,9 @@ export interface HEREMapProps extends H.Map.Options {
   hidpi?: boolean,
   interactive?: boolean,
   lg?: Language,
+  /**
+   * @deprecated no longer supported. HERE Maps JS is now bundled with this package.
+   */
```

Comment:
As this is a breaking change, should we remove all deprecated options?

Reply:
That makes sense. At first I was hoping not to make it a breaking change, but I had to change the default engine which has to be a breaking change.